tools/image_processing/yolo/yolo_training.xml

```diff
@@ -0,0 +1,148 @@
+<tool id="yolo_training" name="Perform YOLO training" version="@VERSION@+galaxy0" profile="24.2">
+    <description>with ultralytics</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <requirements>
+	<container type="docker" >quay.io/sunyi000/ultralytics-galaxy:latest</container>
```

Comment:
should this be moved to the macro?

And it would be nice to give it a version that is somehow indicative